docs/administration/sync-instances/index.md

```diff
@@ -0,0 +1,490 @@
+
+What makes that complex is that items such as lifecycles, environments, accounts, etc., are referred to by id in projects, deployment process steps, and more.  For example, a project has a default lifecycle.  When syncing that project, the process will need to:
+
+1. Translate the lifecycle id on the project to the lifecycle name using data from the source instance.
+2. Translate the lifecycle name to the lifecycle id on the destination instance.
+3. Update the project's default lifecycle id before saving it on the destination instance.
+
+That complexity is further exacerbated by the fact that some data is required, for example, a project's default lifecycle, while other data is not, for example, a step scoped to an environment.
+
+### Data that must be an exact match
+
+The following items must be an exact match between your instances.  If not you will get either missing data errors, corrupted projects, 400 bad requests or unexpected runs from deployments or runbook runs.
+
+- Script Modules
+- Step Templates
+- Tenant Tags
+
+### Data with the same name but different details
+
+Most of the data referenced by your deployment and runbook processes have to exist.  For example, a deployment process directly references a worker pool with the name **Ubuntu Worker Pool**. While the data has to exist, only the name must be the same between instances.  The source instance could have 5 EC2 instances for the **Ubuntu Worker Pool**, while the destination instance could have 3 EC2 instances in a different region.  As long as the worker pool **Ubuntu Worker Pool** exists in both instances with running workers, everything will work fine.
+
+The data that must exist but can have different details is:
+
+- Infrastructure
+    - Accounts: Same account type, different credentials           
+    - Worker Pools: Different workers
+    - Machine Policies: Different health check frequency, tentacle, and calamari update settings
+- Library
+    - Certificates: same certificate type, different cert
+    - External Feeds: same feed type, different credentials
+    - Lifecycles: different phases and retention policies
+- Server Configuration
+    - Teams: user role mapping, different members
+- Project Groups: You don't have to sync all the projects in a project group; only the project group has to exist.
+
+It is possible to leverage variables instead of directly referencing that data in a deployment or runbook process.  As long as the variable exists, is of the correct type, and is assigned to something that exists on the destination instance, the deployment process will work.  For example, a variable with the name `Project.AWS.Account` is assigned to an account named `Dev/Test Account` on the source instance.  On the destination instance that same variable can be assigned to `Staging/Prod Account`. 
```

Comment:
```suggestion
It is possible to leverage variables instead of directly referencing that data in a deployment or runbook process.  The deployment process will work as long as the variable exists, is of the correct type, and is assigned to something that exists on the destination instance.  For example, a variable named `Project.AWS.Account` refers to an account called `Dev/Test Account` on the source instance.  That same variable can refer to the `Staging/Prod Account` on the destination instance. 
```